### ISSUE TITLE
Source code libPrinterSDK.a

### ISSUE THREAD

**Issue by nguyenthanhtruongntt85** [HeligPfleigh/react-native-thermal-receipt-printer#109]
Where is the source code libPrinterSDK.a?

I use initWithUUIDString
Printer * selectedPrinter = [[Printer init] initWithUUIDString:@"BEC6E843-DA5D-C3C0-18B5-359366820482"];
=> Error: cannot init a class object.

how to use initWithUUIDString?

**Comment by esh-g** [comment]
@HeligPfleigh Yes, Could you please provide a reference to the source of the `libPrinterSDK.a` it would help me a lot.. I have been trying to make my own Printer object and connect directly instead of relying on `getDeviceList` to populate the `_printerArray`...
I hope you reply here to my request... Looking forward to hearing from you

Thank you

**Comment by esh-g** [comment]
@nguyenthanhtruongntt85 Were you able to solve this problem any way?

**Comment by nguyenthanhtruongntt85** [comment]
> @nguyenthanhtruongntt85 Were you able to solve this problem any way?

My app has printer Bluetooth and Wifi.

I Using
Printer Bluetooth iOS: 
 - react-native-bluetooth-escpos-printer-fork
Printer Bluetooth android: 
 - eact-native-thermal-receipt-printer-image-qr

Printer Wifi both Android and iOS: 
 - react-native-thermal-receipt-printer-image-qr
 
 Note:
 - I change many codes of the library because the library has many bugs.
 - This is the problem, not easy to code fast, you can take time 1/2 week or 1 month, and depends on your experience about Bluetooth/wifi printers.
 - This is all file of my fix library.
 
<img width="381" alt="Screen Shot 2023-04-23 at 20 02 00" src="https://user-images.githubusercontent.com/41815920/233841449-dd20b3b9-3464-42e3-a9d6-b8445e8f7940.png">

 I've come from Vietnam.
 Where are you come from?
